src/App.css

```diff
@@ -36,3 +36,35 @@
     transform: rotate(360deg);
   }
 }
+.inputfile {
+	width: 0.1px;
+	height: 0.1px;
+	opacity: 0;
+	overflow: hidden;
+	position: absolute;
+  z-index: -1;
+  
+}
+.inputfile + label {
+  font-size: 1.25em;
+  font-weight: 700;
+  color: white;
+  background-color: black;
+  display: inline-block;
+}
+
+.inputfile:focus + label,
+.inputfile + label:hover {
+  background-color: red;
+}
+.inputfile + label {
+	cursor: pointer; /* "hand" cursor */
+}
+.inputfile:focus + label {
+	outline: 1px dotted #000;
+	outline: -webkit-focus-ring-color auto 5px;
+}
+
+.opacityLevel {
+  opacity: 0
+}
```

Comment:
Im pretty sure you can do all this using tailwind. but if u want to use ur own custom class u should puut it in tailwind.css and then npm build it.

Reply:
i'm going to delete that for now, I'm going to approach the styling in a different way